conda/recipes/meta.yaml

```
@@ -0,0 +1,53 @@
+package:
+  name: nvjitlink
+  version: 0.1.0
```

Maybe we can template this somehow to ensure the version stays up-to-date with other sources of truth. This is one approach, but maybe there are others worth considering

```suggestion
{% set data = load_file_data("pyproject.toml") %}

package:
  name: nvjitlink
  version: {{ data.get("project", {}).get("version", "0.1.0.dev1") }}
```